sos/report/plugins/sssd.py

```diff
@@ -51,7 +51,8 @@ def setup(self):
         domain = self.collect_cmd_output("sssctl domain-list", pred=sssd_pred)
         if domain['status'] == 0:
             for domain_name in domain['output'].splitlines():
-                self.add_cmd_output("sssctl domain-status -o " + domain_name)
+                self.add_cmd_output("sssctl domain-status " + domain_name)
```

Comment:
A newbie question: can't collecting information from offline domains get stuck?

Reply:
That's a very good question - let me ask the original reporter

Reply:
Original reporter is still testing. I'll ask them to test with the new version just to make sure I haven't done something wrong with the f-strings

Reply:
Reporter tested and confirmed that running that this change doesn't cause sos to get stuck